backend/danswer/llm/answering/prompts/citations_prompt.py

```diff
@@ -0,0 +1,281 @@
+from collections.abc import Callable
```

Comment:
calling these citation_prompt/quote_prompt is a bit misleading as the are not prompt files

Reply:
we have utils.py under `prompts` already—I see this just as a designation that what is in this folder has to do with prompts / creating prompts / managing prompts